hadoop-hdds/rocks-native/pom.xml

```diff
@@ -57,6 +57,11 @@
             <artifactId>mockito-junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
qq: Should this be added to `hdds-common`?

Reply:
this is required only for this module for now. We can look into this later, if other modules require this later